pin/pin.go

```diff
@@ -286,15 +280,13 @@ func (p *pinner) isInternalPin(c cid.Cid) bool {
 
 // IsPinned returns whether or not the given key is pinned
 // and an explanation of why its pinned
-func (p *pinner) IsPinned(c cid.Cid) (string, bool, error) {
-	p.lock.RLock()
-	defer p.lock.RUnlock()
```

Comment:
?

Reply:
ho shoo, I thought it was cascading with `IsPinnedWithType(ctx context.Context, c cid.Cid, mode Mode)` which already has a lock.